plasmapy/formulary/distribution.py

```diff
@@ -264,24 +243,34 @@ def Maxwellian_velocity_2D(
     coeff = (vThSq * np.pi) ** (-1)
     expTerm = np.exp(-vSq / vThSq)
     distFunc = coeff * expTerm
-    if units == "units":
-        return distFunc.to((u.s / u.m) ** 2)
-    elif units == "unitless":
-        return distFunc
-
-
+    return distFunc
+
+
+@validate_quantities(
+    vx={'units': u.m / u.s, 'pass_equivalent_units': True},
+    vy={'units': u.m / u.s, 'pass_equivalent_units': True},
+    vz={'units': u.m / u.s, 'pass_equivalent_units': True},
+    T={
+        'units': u.K, 'equivalencies': u.temperature_energy(),
+        'pass_equivalent_units': True
+      },
+    vx_drift={'units': u.m / u.s, 'pass_equivalent_units': True},
+    vy_drift={'units': u.m / u.s, 'pass_equivalent_units': True},
+    vz_drift={'units': u.m / u.s, 'pass_equivalent_units': True},
+    vTh={'units': u.m / u.s, 'pass_equivalent_units': True},
```

Comment:
Unrelated comment, but: hey @rocco8773, this makes me wonder... what was the rationale on having `pass_equivalent_units` default to `False`?

Reply:
`pass_equivalent_units=True` will allow equivalent units to pass (I know 🙄) and will NOT convert them to be the expected units.  We wanted the default behavior of `validate_quantities` to actually convert the units when possible.  That's the point of the decorator, validate the quantity to what you expect.

Reply:
Ooooooh. So we don't need these here at all, right?

Reply:
Correct! In fact, you don't even need to include `vx`, `vy`, and `vz` in the validator since the validator is automatically triggered by the typing annotation.

Reply:
You can do something more along the lines of...

```python
@validate_quantities(T={'units': u.K, 'equivalencies': u.temperature_energy()})
def Maxwellian_velocity_3D(
    vx: u.m / u.s,
    vy: u.m / u.s,
    vz: u.m / u.s,
    T: u.K,
    particle="e",
    particle="e",
    vx_drift: u.m / u.s = 0. * u.m / u.s,
    vy_drift: u.m / u.s = 0. * u.m / u.s,
    vz_drift: u.m / u.s = 0. * u.m / u.s,
    vTh: u.m / u.s = np.nan * u.m / u.s,,
) -> (u.s / u.m) ** 3:
```

Here's the [primer on `validate_quantities`](http://docs.plasmapy.org/en/latest/api/plasmapy.utils.decorators.validate_quantities.html#plasmapy.utils.decorators.validate_quantities)

Reply:
I somehow completely forgot we do it by annotations now! Thanks!

Reply:
Resolving this, reopen if you feel like it :).